tools/unitctl/unitctl/src/unitctl.rs

```diff
@@ -114,6 +114,8 @@ pub(crate) enum Commands {
         )]
         output_format: OutputFormat,
     },
+    #[command(about = "List all configured UNIT applications")]
```

Comment:
```suggestion
    #[command(about = "List all configured Unit applications")]
```

...and we should probably check that we're consistent elsewhere in unitctl.

Reply:
I wouldn't say it's a suggestion!

Unit is _never_ spelled UNIT (it's not an acronym)